addons/rose/addon/styles/rose/components/form/_radio-card.scss

```diff
@@ -75,7 +75,6 @@ $radio-svg: (
   }
   .rose-form-radio-label {
     @include type.type(s, bold);
-    display: flex;
     color: var(--black);
```

Comment:
We shouldn't use absolute colors like this except in very specific cases.  Use relative poles instead (stark/subtle).

Reply:
Wow my eyes glazed right over that, I removed it and it looks like it defaults to the body's color of `var(--stark)`, should this be explicit for the `color` and `background-color` or is it ok to inherit it?

Reply:
I prefer inherited.  Less code.